### ISSUE TITLE
High Contrast theme

### ISSUE THREAD

**Issue by aichi** [microsoft/fluent-ui-react#290]
### All application sold in US should be A11y ready
Please provide high contrast theme for all components available in Stardust.

**Comment by alinais** [comment]
@aichi, we have already a planned item that will take care of updating the high contrast mode theme. Stay tunned!

**Comment by alinais** [comment]
covered by #349
